packages/nc-gui/components/smartsheet/toolbar/ViewActions.vue

```diff
@@ -46,8 +48,8 @@ const { isUIAllowed } = useUIPermission()
 
 const { isSharedBase } = useProject()
 
-const Icon = computed(() => {
-  switch (selectedView?.value.lock_type) {
+const Icon = $computed(() => {
+  switch (selectedView.value?.lock_type) {
     case LockType.Personal:
```

Comment:
no need to convert it into `$computed` since it's only getting used in the template